### PR TITLE
Disable brush in merge mode

### LINKED ISSUE #4757: Disable brush in hybrid tracings when merge mode is activated
Right now, the merge moder doesn't listen to segmentation updates. Let's disable the brush for now to avoid inconsistencies.

### PR DESCRIPTION
**Pull request by grittaweisheit**
This PR disables the brush when merger mode is enabled in a hybrid tracing.

### URL of deployed dev instance (used for testing):
- https://disablebrushinmergemode.webknossos.xyz

### Steps to test:
- open a hybrid tracing
- select the brush tool
- enable merger mode
- the tool should change to "move" and the brush button should be disabled
- disable merger mode
- the brush should be available again

### Issues:
- fixes #4757

------
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] ~~Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable~~
- [ ] ~~Updated [documentation](../blob/master/docs) if applicable~~
- [ ] ~~Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes~~
- [ ] ~~Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

File: frontend/javascripts/oxalis/view/action-bar/volume_actions_view.js [hunk 1]
```diff
@@ -17,9 +17,16 @@ const ButtonGroup = Button.Group;
 
 type Props = {|
   activeTool: VolumeTool,
+  isInMergerMode: boolean,
 |};
 
 class VolumeActionsView extends PureComponent<Props> {
+  componentDidUpdate = (preProps: Props) => {
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/view/action-bar/volume_actions_view.js:24]:
```suggestion
  componentDidUpdate = (prevProps: Props) => {
```

It's usually called `prevProps` - meaning previous props :)

File: frontend/javascripts/oxalis/view/action-bar/volume_actions_view.js [hunk 2]
```diff
@@ -17,9 +17,16 @@ const ButtonGroup = Button.Group;
...
+    if (!preProps.isInMergerMode && this.props.isInMergerMode) {
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/view/action-bar/volume_actions_view.js:25]:
```suggestion
    if (!prevProps.isInMergerMode && this.props.isInMergerMode) {
```

File: frontend/javascripts/oxalis/view/action-bar/volume_actions_view.js [hunk 3]
```diff
@@ -42,7 +49,9 @@ class VolumeActionsView extends PureComponent<Props> {
         >
           <RadioButton value={VolumeToolEnum.MOVE}>Move</RadioButton>
           <RadioButton value={VolumeToolEnum.TRACE}>Trace</RadioButton>
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/view/action-bar/volume_actions_view.js:51]: The TRACE tool can also be used to volume annotate. It should be disabled in the same way.

**Comment by daniel-wer** [comment]
I noticed that the button disabled style was broken for the dark theme header (not your fault!), because we overwrote some color values there and fixed that quickly, hope that is alright with you. Now disabled buttons are grayed out as usual and can be recognized without hovering :)

**Comment by grittaweisheit** [comment]
Thanks a lot for fixing the button style!

I added the tooltip and disabled the trace tool as well :)

File: CHANGELOG.unreleased.md [hunk 4]
```diff
@@ -25,6 +25,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.released
 - In the Edit/Import Dataset form, the "Sharing" tab was renamed to "Sharing & Permissions". Also, existing permission-related settings were moved to that tab.  [#4683](https://github.com/scalableminds/webknossos/pull/4763)
 - Improved rotation of camera in 3D viewport. [#4768](https://github.com/scalableminds/webknossos/pull/4768)
 - Improved handling and communication of failures during download of data from datasets. [#4765](https://github.com/scalableminds/webknossos/pull/4765)
+- The annotation tools in hybrid tracings are disabled while in merger mode. [#4757](https://github.com/scalableminds/webknossos/pull/4770)
```
Comment (daniel-wer) [inline CHANGELOG.unreleased.md:28]:
```suggestion
- The volume annotation tools in hybrid tracings are disabled while in merger mode. [#4757](https://github.com/scalableminds/webknossos/pull/4770)
```

**Review by daniel-wer — APPROVED** [review]
Very nice, one small suggestion and then LGTM :+1:

**Pull request merged**